build.go

```diff
@@ -190,13 +190,12 @@ func (Test) Lint() error {
 	mg.Deps(Build.BuildContainer)
 	fmt.Println("\n=====> Linting Satellite...\n")
 	return trace.Wrap(sh.RunV(
-		"docker", "run", "-it", "--rm=true",
+		"docker", "run", "-t", "--rm=true",
 		fmt.Sprintf("--volume=%v:/go/src/github.com/gravitational/satellite", srcDir()),
 		`--env="GOCACHE=/go/src/github.com/gravitational/satellite/build/cache/go"`,
 		fmt.Sprint("satellite-build:", version()),
-		"bash", "-c",
-		"cd /go/src/github.com/gravitational/satellite; golangci-lint run -n --deadline=30m --enable-all"+
-			" -D gochecknoglobals -D gochecknoinits",
+		"golangci-lint", "run", "--new", "--deadline=30m", "--enable-all",
```

Comment:
Not sure why the `bash`/`cd` was required - the Dockerfile specifies it as `WORKINGDIR`.